editor/src/clj/editor/app_view.clj

```diff
@@ -264,29 +264,28 @@
 
     (when is-empty
       (let [label-font (Font. "Dejavu Sans Mono" 13)
-            key-font (Font. "" 13)
-            color (Color. 1.0 1.0 0.59765625 0.6)]
```

Comment:
Kinda weird, after the update, the text "Open Asset" etc that you see when no tab is open suddenly became yellow. Perhaps javafx changed priorities of directly applied color (as it was here) vs colors from styles so it behaves as intended, but I don't think we intend to use some weird yellow colors.